CONTRIBUTING.md

```diff
@@ -62,31 +63,15 @@ These are the tools used in `dbt-common` development and testing:
 
 A deep understanding of these tools in not required to effectively contribute to `dbt-common`, but we recommend checking out the attached documentation if you're interested in learning more about each one.
 
-#### Virtual environments
-
-We strongly recommend using virtual environments when developing code in `dbt-common`. We recommend creating this virtualenv
-in the root of the `dbt-common` repository. To create a new virtualenv, run:
-```sh
-python3 -m venv env
-source env/bin/activate
-```
-
-This will create and activate a new Python virtual environment.
-
 ## Running `dbt-common` in development
 
 ### Installation
 
-First make sure that you set up your `virtualenv` as described in [Setting up an environment](#setting-up-an-environment).  Also ensure you have the latest version of pip installed with `pip install --upgrade pip`. Next, install `dbt-common` (and its dependencies):
-
-```sh
-git
-pre-commit install
-```
+Ensure you have the latest version of pip installed with `pip install --upgrade pip` as well as [hatch](https://hatch.pypa.io/latest/install/).
 
 ### Running `dbt-common`
 
-This repository is just a template and cannot be run.
+This repository cannot be run on its own.
```

Comment:
I'm not sure that this is entirely true. I think it just has very specific use cases that make it less valuable outside of the dbt ecosystem. I'm arguing semantics, but I think the distinction we're looking for is that between a library and an application. `dbt-core` was more of an application whereas this is a library. I'm only pointing it out because this may convey a different sentiment than intended. After all, one of the goals of breaking up the monorepo was to establish stand-alone packages, albeit very use-case-specific packages.